scripts/createVersionJSON.sh

```diff
@@ -52,12 +52,13 @@ function getBranch {
 }
 
 function toJSON {
-    local tpl='{ "version": "#version", "commit": "#commit", "branch": "#branch", "date": "#date" }';
+    local tpl='{ "version": "#version", "commit": "#commit", "branch": "#branch", "buildDate": "#date", "release": "#release" }';
     local commit=$(getCommit);
     local version=$(getVersion);
     local branch=$(getBranch);
-    local date="$(date '+%s')";
-    echo "$tpl" | sed "s/#commit/$commit/; s/#version/$version/; s|#branch|$branch|; s|#date|$date|;"
+    local date="$(date '+%FT%T%Z')";
+    local release="$(git describe --long --dirty --all)";
+    echo "$tpl" | sed "s/#commit/$commit/; s/#version/$version/; s|#branch|$branch|; s|#date|$date|; s|#release|$release|;"
```

Comment:
some of these use `/` and others use `|`. I think I know why. One safer (yet a bit more complicated) alternative is `envsubst`. Here are a couple of examples:

https://github.com/ProtonMail/Angular/blob/v4/docker/entrypoint.sh#L9
https://gitlab.protontech.ch/proton/devops/test-infra/blob/master/kube-get-current-versions.sh#L26

Reply:
I  wasn't able to make it run :/ so instead I use cat :grin:

Reply:
🆒